src/components/Send/direct-send/views/Initial.direct.send.view.tsx

```
@@ -0,0 +1,218 @@
+    })
+    const [showSuccess, setShowSuccess] = useState(false)
+    const [successTxHash, setSuccessTxHash] = useState('')
+
+    const { selectedWallet, signInModal, isConnected, wallets } = useWallet()
+
+    const { setLoadingState, loadingState, isLoading } = useContext(loadingStateContext)
+    const { selectedChainID, selectedTokenAddress, selectedTokenDecimals } = useContext(tokenSelectorContext)
+
+    const { prepareDirectSendTx, sendTransactions, estimateGasFee } = useCreateLink()
+
+    const peanutWallet = wallets && wallets.find((wallet) => wallet.walletProviderType === WalletProviderType.PEANUT)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)